spec/index.md

```diff
@@ -1361,11 +1365,11 @@ The following sections describe various System API functions, also referred to a
 
     ic0.certified_data_set : (src: i32, size: i32) -> ();                       // I G U Ry Rt T
     ic0.data_certificate_present : () -> i32;                                   // *
-    ic0.data_certificate_size : () -> i32;                                      // Q CQ
-    ic0.data_certificate_copy : (dst: i32, offset: i32, size: i32) -> ();       // Q CQ
+    ic0.data_certificate_size : () -> i32;                                      // NRQ CQ
+    ic0.data_certificate_copy : (dst: i32, offset: i32, size: i32) -> ();       // NRQ CQ
 
     ic0.time : () -> (timestamp : i64);                                         // *
-    ic0.global_timer_set : (timestamp : i64) -> i64;                            // I G U Ry Rt C T
+    ic0.global_timer_set : (timestamp : i64) -> i64;                            // I G U RQ Ry Rt C T
```

Comment:
This isn't implemented yet, right?

Reply:
Also it might have unexpected consequences as e.g., the Rust timers CDK needs to alter state when registering a new timer and these state changes will be rolled back in replicated query mode.

Reply:
No, and I was actually unsure whether we should do it. It just felt that there's no strong reason to disallow using this API in replicated queries but indeed it's not going to be usable by the Rust CDK so maybe we I revert this for now.

Reply:
Reverted this change.